php-exercise1/src/model/model.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace model;
+
+class Message{
+    function showMessage(){
```

Comment:
Please refer to PSR-1 and PSR-12 for best practices when writing PHP scripts.

Reply:
Done